rundeckapp/grails-app/i18n/messages.properties

```diff
@@ -44,6 +44,7 @@ default.paginate.prev=-
 default.paginate.next=+
 default.paginate.rew=«
 default.paginate.ff=»
+action.take.amount.time=This action could take a significant amount of time
```

Comment:
I think this is a more appropriate message
"This action might take a while"

Reply:
Done...